spec/controllers/reservations_controller_spec.rb

```diff
@@ -603,7 +603,8 @@
       context 'next reservation is between 5 minutes' do
         let(:next_reservation) do
           Reservation.new :reserve_start_at => Time.zone.parse('2013-08-15 12:02'),
-                          :reserve_end_at   => Time.zone.parse('2013-08-15 12:17')
+                          :reserve_end_at   => Time.zone.parse('2013-08-15 12:17'),
```

Comment:
Can we start switching to 1.9 hash syntax as we're working on new stuff? No need for a full-out campaign, but we should change it where we're touching the code.